src/gui/sdlmain.cpp

```diff
@@ -4532,7 +4532,7 @@ int sdl_main(int argc, char *argv[])
 
 	loguru::init(argc, argv);
 
-	LOG_MSG("dosbox-staging version %s", DOSBOX_GetDetailedVersion());
+	LOG_MSG("%s version %s", CANONICAL_PROJECT_NAME, DOSBOX_GetDetailedVersion());
```

Comment:
Same as above (to be displayed as `DOSBox Staging`). Probably some other places too as long as they are displayed to users.

Reply:
Many unix programs use the lower-case and string-parseable name in their version record - but will also use their public name in a verbose block.

This is usually because scripts and scanner will mass-parse all program `--version` results, and create inventories and such.  This can certainly be the case, so I'm reluctant to change this (even aside from the scope concern above).

Reply:
For example, GCC's official name is `GCC, the GNU Compiler Collection`, however the last line of `gcc --version`:

`gcc version 12.1.0 (Ubuntu 12.1.0-2ubuntu1~22.04) `

Reply:
The original suggestion was made in consistent with previous efforts such as https://github.com/dosbox-staging/dosbox-staging/issues/1803

If there are other concerns (PR scope etc), perhaps they can be kept as is for the purpose of this PR.